src/main/java/com/teragrep/cfe18/CaptureGroupMapper.java

```diff
@@ -53,14 +53,17 @@
 @Mapper
 public interface CaptureGroupMapper {
 
-    List<CaptureGroup> getCaptureGroupByName(String capture_def_group_name,Integer version);
+    CaptureGroup create(String groupName, CaptureGroup.groupType type);
 
-    CaptureGroup addNewCaptureGroup(
-            String capture_def_group_name,
-            Integer capture_definition_id
-    );
+    CaptureGroup createLink(int captureId, int id);
 
-    List<CaptureGroup> getAllCaptureGroup(Integer version);
+    CaptureGroup get(int id,Integer version);
 
-    CaptureGroup deleteCaptureGroup(String name);
+    List<CaptureGroup> getCaptures(int id,Integer version);
+
+    List<CaptureGroup> getAll(Integer version);
+
+    CaptureGroup delete(int id);
+
+    CaptureGroup deleteCapture(int captureId, int id);
```

Comment:
deleteCapture is unlink operation, not delete operation in api perspective, please rename